README.md

```diff
@@ -34,6 +33,13 @@ $ cp .env.template .env  # (first time only)
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like development mode (which also enables features like hot reloading when you make a file change). There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
 
+Replace the following secrets to the env list before starting the app example:
+```bash
+TRELLO_KEY=123123123
+TROLLO_TOKEN=1231231123
```

Comment:
Thanks for adding clear instructions for these. It would be nice to also add them to the `.env.template` file so that they're also obvious there when setting up, and note the (_maybe_ intentional?) typo in your token & board id! (Which you'll want to adjust in the code as well!)
```suggestion
TRELLO_TOKEN=1231231123
```